test/unit/MatchingEngine.spec.ts

```diff
@@ -28,6 +32,11 @@ peerPubKey = '029a96c975d301c1c8787fcb4647b5be65a3b8d8a70153ff72e3eac73759e5e345
   pairId: PAIR_ID,
 });
 
+const isEmpty = (engine: MatchingEngine) => {
+  expect(engine.ownOrders.sell).to.be.empty;
+  expect(engine.queues.sell.isEmpty()).to.be.true;
```

Comment:
Why are you just checking the sell queue and list and not the buy ones?

Reply:
Good catch, it was dropped. I'll add it.